tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<blockquote>Checkpointing cannot be done with Keras HDF5 weights.</blockquote><p>This is not true for weight clustering yet. It's only the case for pruning because we have a UpdatePruningStep function and the PruningSchedule relies on the step to saved and restored. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>